README.rdoc

```diff
@@ -38,5 +38,10 @@ In your Gemfile:
 
   gem 'ruby-hl7'
 
+=== Migrating from 1.2.3 to 1.2.4
```

Comment:
Thank you so much for adding this  💖 !!

Since this is a change that might affect people using `1.2.3` I'm thinking the next release should be `1.3.0`

Reply:
Hahah, wrong guess 🙂 
I just fixed it.